<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="return_Button">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>450</y>
      <width>92</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_Button">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>490</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>450</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="id">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>学号：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="functionBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>450</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>选择操作</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>增加</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>删除</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>修改</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>查询</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="nameLine">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>450</y>
      <width>241</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="idLine">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>490</y>
      <width>241</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="confirm_Button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>490</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>681</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="genderBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>530</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>选择性别</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>男</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>女</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="gender">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>530</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>性别：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
